config.js

```diff
@@ -52,7 +52,7 @@ const ConfigOptions = () => {
          */
         importedAssets: {
             importedAssetsDir: 'imported-assets',
-            importedAssetsSrcGlob: 'node_modules/@justeat/**/*'
+            importedAssetsSrcGlob: 'node_modules/@justeat/*/'
```

Comment:
Yep, to match only directories ([see options.nodir](https://github.com/isaacs/node-glob#options))